src/codemagic/tools/_app_store_connect/action_groups/builds_action_group.py

```diff
@@ -39,3 +53,143 @@ def get_build_pre_release_version(self, build_id: ResourceId, should_print: bool
             self.api_client.builds.read_pre_release_version,
             should_print,
         )
+
+    @cli.action(
+        'add-beta-test-info',
+        BuildArgument.BUILD_ID_RESOURCE_ID,
+        BuildArgument.BETA_BUILD_LOCALIZATIONS,
+        BuildArgument.LOCALE_DEFAULT,
+        BuildArgument.WHATS_NEW,
+        action_group=AppStoreConnectActionGroup.BUILDS)
+    def add_beta_test_info(self,
+                           build_id: ResourceId,
+                           beta_build_localizations: Optional[Types.BetaBuildLocalizations] = None,
+                           locale: Optional[Locale] = None,
+                           whats_new: Optional[Types.WhatsNewArgument] = None):
+        """
+        Add localized What's new (what to test) information
+        """
+
+        beta_test_info_items = [*beta_build_localizations.value] if beta_build_localizations else []
+
+        if whats_new:
+            beta_test_info_items.append(BetaBuildInfo(whats_new=whats_new.value, locale=locale))
+
+        self.logger.info(Colors.BLUE('\nUpdate beta build localization info in TestFlight for uploaded build'))
+        for item in beta_test_info_items:
+            self.create_beta_build_localization(build_id=build_id, locale=item.locale, whats_new=item.whats_new)
+
+    @cli.action(
+        'submit-to-testflight',
+        BuildArgument.BUILD_ID_RESOURCE_ID,
+        action_group=AppStoreConnectActionGroup.BUILDS)
+    def submit_to_testflight(
+            self,
+            build_id: ResourceId,
+            max_build_processing_wait: Optional[Union[int, Types.MaxBuildProcessingWait]] = None):
+        """
+        Submit build to TestFlight
+        """
+
+        # Workaround to support overriding default value by environment variable.
+        if max_build_processing_wait is not None:
+            max_processing_minutes = max_build_processing_wait.value if \
+                isinstance(max_build_processing_wait, Types.MaxBuildProcessingWait) else max_build_processing_wait
+        else:
+            max_processing_minutes = Types.MaxBuildProcessingWait.default_value
+
+        build, app = self.api_client.builds.read_with_include(build_id, App)
+
+        self._assert_app_has_testflight_information(app)
+
+        if max_processing_minutes:
```

Comment:
As `max_processing_minutes` does more than it used to, perhaps it makes sense to update docs for this argument too that in case of `0` value build processing check will be omitted?